### PR TITLE
Qml marketplace - multiple improvements/bugfixes

### PR DESCRIPTION
**Pull request by roxanneskelly**
https://highfidelity.manuscript.com/f/cases/21118/descending-sorts-don-t-work-in-new-qml-marketplace

https://highfidelity.manuscript.com/f/cases/21119/text-filter-doesn-t-change-as-you-type-in-new-qml-marketplace

https://highfidelity.manuscript.com/f/cases/21120/text-filter-doesn-t-clear-in-new-qml-marketplace

https://highfidelity.manuscript.com/f/cases/21123/marketplace-license-display-has-weird-crud-in-it-new-qml-version

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17233/HighFidelity-Beta-PR14906-a82221d.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17233/HighFidelity-Beta-PR14906-a82221d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17233/HighFidelity-Beta-Interface-PR14906-a82221d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17233/HighFidelity-Beta-PR14906-a82221d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17233/HighFidelity-Beta-Interface-PR14906-a82221d.exe

File: interface/resources/qml/hifi/commerce/marketplace/MarketplaceItem.qml [hunk 1]
```diff
@@ -520,7 +520,7 @@ Rectangle {
                         } else if (root.license === "Attribution-NonCommercial-NoDerivs (CC BY-NC-ND)") {
                           url = "https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.txt"
                         } else if (root.license === "Proof of Provenance License (PoP License)") {
-                          url = "https://digitalassetregistry.com/PoP-License/v1/"
+                          url = "licenses/Popv1.txt"
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/commerce/marketplace/MarketplaceItem.qml]: .txt? Interesting. (I can guess.....)

Comment (roxanneskelly) [reply interface/resources/qml/hifi/commerce/marketplace/MarketplaceItem.qml]: while it contains some markup, it's not a full html document so I just made it a text file.

**Comment by rickcarr044** [comment]
Currently testing PR14906

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17327/HighFidelity-Beta-PR14906-a82221d.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17327/HighFidelity-Beta-PR14906-a82221d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17327/HighFidelity-Beta-Interface-PR14906-a82221d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17327/HighFidelity-Beta-PR14906-a82221d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17327/HighFidelity-Beta-Interface-PR14906-a82221d.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17335/HighFidelity-Beta-PR14906-1154414.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17335/HighFidelity-Beta-PR14906-1154414.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17335/HighFidelity-Beta-Interface-PR14906-1154414.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17335/HighFidelity-Beta-PR14906-1154414.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17335/HighFidelity-Beta-Interface-PR14906-1154414.exe

**Pull request merged**